pkg/controller/operators/catalog/operator.go

```diff
@@ -528,6 +531,26 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 					plan.Status.Plan[i].Status = v1alpha1.StepStatusCreated
 				}
 
+			case clusterRoleKind:
+				// Marshal the manifest into a ClusterRole instance.
+				var cr rbacv1.ClusterRole
+				err := json.Unmarshal([]byte(step.Resource.Manifest), &cr)
+				if err != nil {
+					return err
+				}
+
+				// Attempt to create the ClusterRole.
+				_, err = o.OpClient.KubernetesInterface().RbacV1().ClusterRoles().Create(&cr)
```

Comment:
Is it possible that the cluster role is ever modified? If so, perhaps reconciliation should be used here instead of create?

Reply:
It could be modified, but the InstallPlans are "run-once". So you just run a new installplan and it will re-create if needed.